src/pycmor/core/pipeline.py

```diff
@@ -246,8 +246,11 @@ class DefaultPipeline(FrozenPipeline):
 
     Notes
     -----
-    The pipeline includes automatic vertical bounds calculation for datasets with vertical coordinates
-    (pressure levels, depth, height), ensuring CMIP compliance.
+    The pipeline includes:
+    - Automatic vertical bounds calculation for datasets with vertical coordinates (pressure levels, depth, height)
+    - Dimension mapping from source data to CMIP dimension names (e.g., 'latitude' → 'lat', 'lev' → 'plev19')
```

Comment:
you might need to wrap these in backticks to get it to render correctly:


```suggestion
    - Dimension mapping from source data to CMIP dimension names (e.g., ``'latitude'`` → ``'lat'``, ``'lev'`` → ``'plev19'``)
```